pkgs/applications/editors/neovim/neovide/default.nix

```diff
@@ -92,9 +84,9 @@ rustPlatform.buildRustPackage rec {
   ];
 
   postFixup = ''
-    wrapProgram $out/bin/neovide \
-      --prefix LD_LIBRARY_PATH : ${vulkan-loader}/lib
-  '';
+      wrapProgram $out/bin/neovide \
+        --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libglvnd xorg.libXcursor xorg.libXext xorg.libXrandr xorg.libXi ]}
```

Comment:
I also added `wayland` and `libxkbcommon` to allow Neovide to run without XWayland on Sway.